dynamodb/src/main/scala/zio/dynamodb/DynamoDBExecutorImpl.scala

```diff
@@ -109,17 +113,17 @@ private[dynamodb] final case class DynamoDBExecutorImpl private (dynamoDb: Dynam
       case ZIOAwsTableStatus.INACCESSIBLE_ENCRYPTION_CREDENTIALS => TableStatus.InaccessibleEncryptionCredentials
       case ZIOAwsTableStatus.ARCHIVING                           => TableStatus.Archiving
       case ZIOAwsTableStatus.ARCHIVED                            => TableStatus.Archived
-      case ZIOAwsTableStatus.unknownToSdkVersion                 => ??? // What to do about this one?
+      case ZIOAwsTableStatus.unknownToSdkVersion                 => TableStatus.unknownToSdkVersion
     }
 
   private def generateDeleteItemRequest(deleteItem: DeleteItem): DeleteItemRequest =
     DeleteItemRequest(
       tableName = deleteItem.tableName.value,
-      key = deleteItem.key.map.map { case (k, v) => (k, buildAwsAttributeValue(v)) },
+      key = deleteItem.key.map.map { case (k, v) => (k, DynamoDBExecutorImpl.buildAwsAttributeValue(v)) },
```

Comment:
just curious why the `DynamoDBExecutorImpl` prefix was introduced?

Reply:
I added it to hold some of the conversion functions from our model to the zio-aws model. I am using them in the tests in this PR but am wondering if that's a good idea now or if I should just create the zio-aws object and pass it instead.

Reply:
np, I was just asking if `DynamoDBExecutorImpl` was not imported for clarity - which I guess is the case so that's cool

Reply:
I imported the object, not sure why I didn't do this the first time.